app/models/user.rb

```diff
@@ -1,11 +1,13 @@
 class User < ApplicationRecord
   has_secure_password
 
-  has_and_belongs_to_many :pairings
```

Comment:
rubocop did not approve of this. After further Googling, a has_many through is better convention and more future-proof